.github/ISSUE_TEMPLATE/learn-feedback.yml

```diff
@@ -0,0 +1,68 @@
+name: learn.microsoft.com feedback control
+description: |
+  ⛔ This template is integrated into the learn.microsoft.com feedback control on the bottom of each API reference page. Several fields are automatically filled. Don't use for other purposes. ⛔
+labels:
```

Comment:
do we need to assign any triage label or owners? Who is going to be looking at these?

Reply:
These will ALWAYS be docs-related issues, which is why I'm confident we can automatically apply the "Docs" label. It may make sense to also apply the "needs-team-triage" label. Thoughts?

The author of the doc will be listed in the issue. For example, Karishma would be tagged if an issue is created for this page: https://learn.microsoft.com/javascript/api/overview/azure/identity-readme?view=azure-node-latest. That's because her GitHub alias appears in the `ms.author` metadata field for that page. Whoever triages the issues will need to manually assign the appropriate person.

Reply:
Gotcha, yeah I think `needs-team-triage` would be good

Reply:
Please don't add labels here.   The automation will detect when an issue is opened and run the workflow that does triage.   It keys off of labels as well as state and adding things outside of the bot could break things.